src/main/java/Event.java

```diff
@@ -0,0 +1,26 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected String dateNum;
+    protected LocalDate date;
+
+    public Event(String description, String date) {
+        super(description);
+        this.dateNum = date;
+        this.date = LocalDate.parse(date);
+    }
+
+    @Override
+    public String getTask() {
+        return "E";
```

Comment:
Any reason why you did it this way instead of using a constant or final variable? Same for a few other places...